src/source/PeerConnection/Rtcp.c

```diff
@@ -1,25 +1,81 @@
 #define LOG_CLASS "RtcRtcp"
 
 #include "../Include_i.h"
+#include "kvsrtcp/rtcp_api.h"
+
+static RtcpPacketType_t getDetailedRtcpPacketType(uint8_t packetType, uint8_t receptionReportCount)
+{
+    RtcpPacketType_t result = RTCP_PACKET_UNKNOWN;
+
+    switch (packetType) {
+        case RTCP_PACKET_TYPE_FIR: {
+            if (receptionReportCount == 0) {
+                result = RTCP_PACKET_FIR;
```

Comment:
return (receptionReportCount == 0) ? RTCP_PACKET_FIR : RTCP_PACKET_UNKNOWN;
has more clarity on the logic. .assumes that the packetType is always valid is that true?

Reply:
result = RTCP_PACKET_UNKNOWN is he default value, hence if any of the swtich case is not met, its by default RTCP_PACKET_UNKNOWN. Hence i think  "(receptionReportCount == 0) ? RTCP_PACKET_FIR : RTCP_PACKET_UNKNOWN;" this might not be needed.